client/src/forms/Autocomplete.res

```diff
@@ -537,17 +537,18 @@ let tlDestinations = (m: model): list<autocompleteItem> => {
 // Create the list
 // ------------------------------------
 
-let allowedParamTipes = /* Types from Types.tipe that aren't included:
-       TCharacter - very hard to create characters right now, so this type
-                    would raises more questions than answers right now
-       TNull - trying to get rid of this, so don't spread it
-       TIncomplete - makes no sense to pass to a function
-       TError - makes no sense to pass to a function
-       TResp - these aren't really exposed to users as real things, but maybe
-               should be?
-       TErrorRail  - doesn't make sense pass to function
-       TDbList - only for DB schemas
-       TUserType  - added later
+let allowedParamTipes =
+/* Types from Types.tipe that aren't included:
+TCharacter - very hard to create characters right now, so this type
+            would raises more questions than answers right now
+TNull - trying to get rid of this, so don't spread it
+TIncomplete - makes no sense to pass to a function
+TError - makes no sense to pass to a function
```

Comment:
<img width="392" alt="image" src="https://user-images.githubusercontent.com/181762/177006253-b9ae5678-f344-464c-8988-6820da370652.png">

<img width="419" alt="image" src="https://user-images.githubusercontent.com/181762/177006261-230e3a05-0e69-4510-abf3-1a29097b3bf5.png">

I think this needs code to parse this (and tests!). I think adding tuples to the interestingDvals should find this.

Reply:
I also get an error in Blazor when calling functions with tuples of 3 or more iterms:


<img width="1419" alt="image" src="https://user-images.githubusercontent.com/181762/177006322-ede7770a-c900-4140-a218-a0750e706a27.png">

<img width="458" alt="image" src="https://user-images.githubusercontent.com/181762/177006333-bbf96359-887e-4b36-9d7b-d2703046f695.png">

Reply:
I think we shouldn't support tuple-typed function parameters until we support polymorphic type signatures. With (I think) all the existing types, such as `TList`, there's an easy conversion from the old value to the new one: `Tlist` becomes `TList TAny`. However, in TTuple, we encode the number of type arguments as well as their actual argument. So if someone has a `TTuple`, and they call it with a tuple with 7 members, how will we know to convert it to `TTuple(TAny,TAny,TAny,TAny,TAny,TAny,TAny)` instead of `TTuple(TAny,TAny)`.

So this makes me think we should remove the ability to call functions with tuples for now.

Reply:
Agreed - I extracted this and type checking to https://github.com/darklang/dark/issues/4239